application/controllers/classification_type_controller.py

```diff
@@ -52,3 +52,26 @@ def get_classification_types():
         return [classification.to_dict() for classification in classification_types]
 
     logger.info("Attempted to retrieve classification types when none in database")
+
+
+def _delete_classification_type(classification_type):
```

Comment:
Would prefer the name not to just be a mirror of the controller, with the "_" added, something like _query_delete_classification might be better.